src/test/kotlin/com/terraformation/backend/seedbank/db/AccessionStoreTest.kt

```diff
@@ -318,6 +331,25 @@ internal class AccessionStoreTest : DatabaseTest(), RunsAsUser {
     assertEquals(newSpeciesName, initial.species, "Species name")
   }
 
+  @Test
+  fun `create inserts quantity history row if quantity is specified`() {
```

Comment:
another test for when quantity is not specified - to check that there's new new history entry? same for the update scenario.